ksql-engine/src/main/java/io/confluent/ksql/function/udaf/count/CountKudaf.java

```diff
@@ -37,6 +37,9 @@
 
   @Override
   public Long aggregate(final Object currentValue, final Long aggregateValue) {
+    if (currentValue == null) {
+      return aggregateValue;
+    }
```

Comment:
We should count nulls in `count` function. If we need to count non- null values only we can easily add a filter in `WHERE` clause.

Reply:
@hjafarpour My thinking was to emulate what most databases do where `count(*)` returns the count regardless of nulls and `count(<column>)` returns the count where `<column>` is not null.

Reply:
@hjafarpour Do you agree that being consistent with the behavior of most RDBMS's (e.g. Oracle, MySQL, Postgres, SQL Server, DB2), by ignoring `null`, is desirable in this case?

Reply:
I'm in agreement that we should be consistent with the DB world as much as possible